CONTRIBUTING.md

```diff
@@ -34,6 +34,26 @@ _Before_ submitting a pull request, please make sure the following is done…
 
     Note: Replace `<your_username>` with your GitHub username
 
+1.  Run Jest with [Docker](https://docs.docker.com/engine/docker-overview/) (Optional) or continue reading for manual instructions.
```

Comment:
Would it make sense to split this into separate section? "Usage with Docker container" or so?

Adding this step here makes the instructions complex. That might be an issue for some contributors. I mean, currently the instructions are a simple step by step thing. Adding skip this, if you don’t plan using Docker, skip to step 8 now, etc. makes the instructions harder to digest.

For example, one might get stuck thinking: What’s that Docker? Why would I need it?